test-utils/testlib/src/user/rpc_user.rs

```diff
@@ -22,6 +22,7 @@ use near_primitives::views::{
 };
 
 use crate::user::User;
+use near_primitives::rpc::BlockQueryInfo;
```

Comment:
This import should be in the relevant group. *I would prefer CLion would not do this "favor"*